concrete/blocks/event_list/form.php

```diff
@@ -16,29 +16,22 @@
         <label class="control-label"><?= t('Filter By Topic') ?></label>
         <div class="radio">
             <label>
-                <input type="radio" name="filterByTopic" value="none" <?php if ($filterByTopic == 'none') {
-    ?> checked <?php 
-} ?>>
-                No topic filtering.
+                <input type="radio" name="filterByTopic" value="none" <?= $filterByTopic == 'none' ? 'checked' : ''q ?>>
```

Comment:
I guess there's an unwanted `q` here :wink:

Reply:
Friday afternoon... might as well go get a beer instead of coding.